domains/pi19044.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "PrelevatedInsider19044's website",
+    "repo": "https://github.com/pi19044/pi19044.github.io",
+    "owner": {
+    	"username": "pi19044",
+    	"email": "membermadde.insiddev25120@hotmail.com"
+    },
+
+    "record": {
+    	"CNAME": "pi19044.is-a.dev"
```

Uh you can't put CNAME record as the one you are trying to get, maybe you wanted like this?
```suggestion
    	"CNAME": "pi19044.github.io"
```